tools/ci_build/github/azure-pipelines/anybuild.yml

```diff
@@ -0,0 +1,150 @@
+# TODO: update to run nightly or something
+trigger: none
```

Comment:
Btw, I see that "AnyBuild" shows as one of the checks for this PR.  Do you really want to run this for every PR?  I thought the plan was to do it either nightly or as a rolling build (for every new commit in master)?